mypy/semanal_namedtuple.py

```diff
@@ -569,40 +570,46 @@ def add_field(
             add_field(Var("__match_args__", match_args_type), is_initialized_in_class=True)
 
         assert info.tuple_type is not None  # Set by update_tuple_type() above.
-        tvd = TypeVarType(
```

Comment:
What was the name "tvd" supposed to mean?

Glad to see it gone, the only thing I can even come up with is "type var default" but that doesn't seem like the best name for something that's only ever self type even in non-abbreviated form.

Reply:
"tvd" stays for `TypeVarDef` which is not a thing since around couple years ago.